src/components/VectorLogo/Vector.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box } from '@material-ui/core'
+import * as React from 'react'
+
+const Vector: React.FC = () => {
+  return (
+    <Box style={{ display: 'flex', alignItems: 'center' }}>
+      <svg
```

Comment:
just add svg file, why is this a React component?

Reply:
could you answer my question?

Reply:
I thought about putting it in the storybook, but I replaced it with a CardMedia component

Reply:
@mlodyFrontendowiec I think that instead of creating this component you can move svg to its own file and then use it like other svgs that you're using in IDO component; I don't understand why is this not possible in this case

Reply:
I fixed it, yeah you are right my mistake

Reply:
ok, now this component can be removed